tests/sample/reflect_packet.c

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "bpf_helpers.h"
+#include "ebpf.h"
+#include "xdp_tests_common.h"
+
+void
+copy_mac_address(uint8_t destination[6], uint8_t source[6])
```

Comment:
Instead of hardcoding 6 all over the place, either use a `#define` or a typedef (so you can use sizeof() or EBPF_COUNTOF() on it)